Update location_id.yaml

saijam opened the pull request:
Päivitetty Metropolian lokaatiot. Osa koodeista puuttu auktorisoiduista arvoista  esim. Branch METVAR, ei päästä lisäämään, kun CSC-Laurea-Koha on kiinni.

saijam committed:
Update location_id.yaml

Päivitetty Metropolian lokaatiot. Osa koodeista puuttu auktorisoiduista arvoista  esim. Branch METVAR, ei päästä lisäämään, kun CSC-Laurea-Koha on kiinni.

ghost commented:
Tuota CCODEa en määrittelisi noin, jos teillä on tarkoitus käyttää siinä omistajakirjastoa.

saijam committed:
Update location_id.yaml

Päivitetty lokaatiot kokonaisuudessaan

saijam committed:
Päivitetty vielä metrodb eli Branch130

Päivitetty siis Brach130 metroDB konversiosääntöä pari tuntia sitten lähetetystä location.yaml-versiosta.
